src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    img {
+      display: block;
+      object-fit: cover;
+      object-position: center;
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    font-size: inherit;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-tag {
+    color: $secondary;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: block;
+
+    font-size: 14px;
+    font-weight: 500;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+
+    height: 40px;
+    border-radius: 5px;
+    background: $blue-accent;
+    border: 1px solid $blue-accent;
+
+    transition: all .3s ease;
```

Comment:
it's not a good practice to use `all` as transition-property. specify exactly what you'd like to change